frontends/main/src/common/urls.ts

```diff
@@ -1,8 +1,10 @@
+import invariant from "tiny-invariant"
+
 const generatePath = (
   template: string,
   params: Record<string, string | number>,
 ): string => {
-  return template.replace(/:(\w+)/g, (_, key) => {
+  return template.replace(/\[(\w+)\]/g, (_, key) => {
```

Comment:
Marking params as with `[...]` is much more convenient for use with `next-router-mock`... see https://github.com/mitodl/mit-open/pull/1560/files#diff-22d69d0f979fef7f8d9d0b0cd55dd7a00c6c3f4302ac7d542f79220acee4f905R18